includes/sanitizers/class-amp-comments-sanitizer.php

```diff
@@ -172,7 +172,7 @@ protected function process_comment_form( $comment_form ) {
 			sprintf(
 				'submit-success:AMP.setState( { %s: %s } )',
 				$state_id,
-				wp_json_encode( $form_reset_state )
+				wp_json_encode( $form_reset_state, JSON_UNESCAPED_UNICODE )
```

Comment:
@vralle thanks for testing. It looks like I missed this one which I've just added. Please test the new build.

Reply:
Submit button - Ok.
After push "Reply" (Reply to comment) - Comment form title wrong.

Reply:
Can you share the full page source (not from DevTools) in a Gist so I can inspect?

Reply:
I think this is the missing change:

```diff
diff --git a/includes/class-amp-theme-support.php b/includes/class-amp-theme-support.php
index 57afe7c6..a561c76e 100644
--- a/includes/class-amp-theme-support.php
+++ b/includes/class-amp-theme-support.php
@@ -1167,7 +1167,7 @@ class AMP_Theme_Support {
 			str_replace(
 				'%s',
 				sprintf( '" + %s.replyToName + "', $state_id ),
-				wp_json_encode( $args['title_reply_to'] )
+				wp_json_encode( $args['title_reply_to'], JSON_UNESCAPED_UNICODE )
 			),
 			wp_json_encode( $args['title_reply'], JSON_UNESCAPED_UNICODE )
 		);
```